### ISSUE TITLE
A11y_code.visualstudio.com_APIGetStarted _YourFirstExtension_Zoom: Tool tip is not provided for  hamburger menu i.e. "Expand and collapse menu" button at zoom 200%.

### ISSUE THREAD

**Issue by Mohini78** [microsoft/vscode-docs#2792]
“[Check out Accessibility Insights!](https://microsoft.sharepoint.com/teams/msenable/Pages/accessibility-tools.aspx) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”
**Environment Details**:
URL: [https://code.visualstudio.com](https://code.visualstudio.com)
Windows Version: Windows10
Browser Version:
Microsoft Edge 44.18362.1.0
Chrome Version 75.0.3770.100 (Official Build) (64-bit)
Screen Reader Versions : NVDA2019.1.1
MAS Violated:  MAS1.4.4, MAS4.1.2, MAS1.1.1
**Repro Steps**:
Navigate to the URL: [https://code.visualstudio.com/api/get-started/your-first-extension](https://code.visualstudio.com/api/get-started/your-first-extension)
follow below steps:
Launch the application URL: [https://code.visualstudio.com](https://code.visualstudio.com) OR
1.Navigate to "API" link.
2.From left navigation, select "Get Started".
3.Navigate to "Your First Extension" link.
4.Go to browser settings and increase the Zoom to 200%.
5.Using keyboard TAB to the hamburger button and observe the tool tip.

**Actual**:
1.Tool tip is not provided for  hamburger menu i.e. "Expand and collapse menu" button at zoom 200%.
2.Name for the button is inappropriate i.e. "Expand and collapse menu" button.

This issue is observed in Edge, Chrome.

**Expected**:
1.Tool tip must be provided for  hamburger menu i.e. "Expand and collapse menu" button at zoom 200%.
2.Name for the button must be descriptive.

**Recommendations**:
Refer below link which is repository of bug fixes code snippets:
[https://code.visualstudio.com/docs/setup/network](https://code.visualstudio.com/docs/setup/network)

**User Impact**:
Keyboard only users will not be able to understand the context of the button and also screen reader only users will not be able to understand the significance of the button by its name i.e. Expand and Collapse Menu button when the scenario is applied to the setting of Zoom 200%.

People who have difficulty perceiving visual content. Assistive technology can read text aloud, present it visually, or convert it to braille.
Text alternatives or tooltips may help some people who have difficulty understanding the meaning of photographs, drawings, and other images (e.g., line drawings, graphic designs, paintings, three-dimensional representations), graphs, charts, animations, etc. 
People who are deaf, are hard of hearing, or who are having trouble understanding audio information for any reason can read the text presentation. Research is ongoing regarding automatic translation of text into sign language. 
People who are deaf-blind can read the text in braille. 
Additionally, text alternatives support the ability to search for non-text content and to repurpose content in a variety of ways. 

**MAS Reference**:
MAS1.4.4 -
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={b69e5ccc-dde2-4406-b550-4deb38648f6d}](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={b69e5ccc-dde2-4406-b550-4deb38648f6d})
MAS4.1.2- [https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730}](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730})
MAS1.1.1- [https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={d2d2051f-bdc8-4af7-8e18-38aae867e216}](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={d2d2051f-bdc8-4af7-8e18-38aae867e216})

**Attachment for Reference**:
[A11y_code.visualstudio.com_APIGetStarted _YourFirstExtension_5.zip](https://github.com/microsoft/vscode-docs/files/3320321/A11y_code.visualstudio.com_APIGetStarted._YourFirstExtension_5.zip)

**Comment by octref** [comment]
Fixed by updating "Expand and collapse menu" to "Toggle Navigation Menu".
